app/views/news/2018-05-07-april-2018-monthly-update.html.md

```diff
@@ -0,0 +1,66 @@
+---
+summary: In April, we paid for 97 hours of developer work, held a board meeting, and created a new feedback forum for Ruby Together.
+---
+
+Hello! Welcome to the monthly update. During April, our work was supported by [Stripe](https://stripe.com), [Coinbase](https://coinbase.com), [reinteractive](https://reinteractive.com/), [Airbnb](http://airbnb.com), and many others.
+
+
+## ruby together news
+
+In April, Ruby Together was supported by 80 different companies, including Sapphire member [Stripe](https://stripe.com). 6 companies joined as new members, including [Piece of Code](http://www.pieceofcode.com) and [Coinbase](https://www.coinbase.com).
+
+On top of those companies, 6 new developers signed up as members or friends of Ruby Together, including Tom Johnson, Pat Allan, Jameson Hampton, and Jeremy Flores. In total, we were supported by 92 individual members and 71 friends of Ruby Together. Thanks to all of our members for making everything that we do possible. &lt;3
+
+The board met with Ruby developer Sam Phippen to discuss his concerns about Ruby Together’s governance model. As a result of our conversation, we held a board meeting and created a new forum for communication between the community and the board. [We published the results of the meeting](https://rubytogether.org/news/2018-04-25-april-22-board-meeting-results), and we expect to continue addressing community concerns in future work. If you have feedback for the board, or concerns that you’d like to share [open an issue in the Ruby Together [Board GitHub repo](https://github.com/rubytogether/board) or send us an email at [feedback@rubytogether.org](mailto:feedback@rubytogether.org). (And feel free take a look at our [frequently asked questions](https://rubytogether.org/companies#faq) to learn more about how we work.)
```

Comment:
What do you all think about calling out the "new feedback channels" and "new information source repo" out a bit more with a separate heading? It seems important and maybe washes into this paragraph a bit.